src/services/instances.js

```diff
@@ -1,8 +1,9 @@
 import axios from "axios";
-
 export const apiBaseUrl = "https://api.themoviedb.org/3";
 const apiKey = "c33b71412633be3e4a413c17428d1624";
 
+export const imageUrl = "http://image.tmdb.org/t/p/w500/";
```

Comment:
Let's name this `baseImageUrl` indicating it's a partial (base) url, still missing the id for fetching a complete one.